services/agent/src/simcore_service_agent/modules/low_priority_managers.py

```diff
@@ -0,0 +1,30 @@
+from dataclasses import dataclass, field
+from typing import cast
+
+from fastapi import FastAPI
+
+from .concurrency import LowPriorityHandlerManager
+
+
+@dataclass
+class LowPriorityManagers:
```

Comment:
what is a low priority manager?
why is it plural?
why not VolumeCleaner or something similar?